website/docs/d/cis_firewall_rules.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+subcategory: "Internet services"
+layout: "ibm"
+page_title: "IBM: ibm_cis_firewall_rules"
+description: |-
+  Get information on an IBM Cloud Internet Services firewall rules.
+---
+
+# ibm_cis_firewall_rules
+Retrieve information about an existing IBM Cloud Internet Services instance. For more information, see [firewall rule actions](https://cloud.ibm.com/docs/cis?topic=cis-actions).
+
+## Example usage
+
+```terraform
+data "ibm_cis_firewall_rules" "firewall_rules_instance" {
+  cis_id    = ibm_cis.instance.id
+  domain_id = ibm_cis_domain.example.id
+}
+```
+
+## Argument reference
+The following arguments are supported:
+
+- `cis_id` - (Required, String) The ID of the CIS service instance.
+- `domain_id` - (Required, String) The ID of the domain.
+
+## Attributes Reference
+In addition to all arguments above, the following attributes are exported:
+- `id` - (String) The Firewall rules ID. It is a combination of <`firewall_rule_id`>,<`domain_id`>,<`cis_id`> attributes concatenated with ":"
+- `cis_firewallrules_list` - (List of Firewall Rules)
```

Comment:
In schema it is `firewall_rules` not `cis_firewallrules_list`
Please cross check all the attributes with schema